### ISSUE TITLE
Website is out-of-date

### ISSUE THREAD

**Issue by Vampire** [mebigfatguy/fb-contrib#446]
The website https://fb-contrib.sourceforge.net/ says latest version is 7.4.3, while there is already 7.6.0, so I guess there is also other outdated information.

And as the page says

> Please note that active development for this project is now done on github. If this site seems behind, please [check there](http://github.com/mebigfatguy/fb-contrib). I will try to keep this site up to date, but I'd appreciate a poke if you see something missing.

here the poke. :-)

**Comment by mebigfatguy** [comment]
yeah i should really take down  sf. i literally forget how to update the dang thing every time, and have to relearn it.

**Comment by Vampire** [comment]
> i literally forget how to update the dang thing every time

So many ways. :-D
You can use SFTP to upload new files.
Or you can connect to the interactive shell service and do it from there.
...
